style.scss

```diff
@@ -0,0 +1,79 @@
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  background-color: #e5e5e5;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.wrapper {
+  max-width: 375px;
+}
+
+.header {
+  padding: 0 24px;
+  background-color: #fff;
+  background-image: url(./images/header-bg.png);
+  background-repeat: no-repeat;
+  background-position-y: 64px;
+  background-size: contain;
+  border-end-start-radius: 80px;
+}
+
+.toolbar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 48px;
+}
+
+.logo {
+  background-color: #6070ff;
+  padding: 0 10px;
+  border-radius: 3px;
+  color: #fff;
+  font-weight: 600;
+  text-decoration: none;
+}
+
+.headline {
```

Comment:
Please ensure that the content of `headline section` is aligned with the flexbox